devimint/src/external.rs

```diff
@@ -554,6 +554,22 @@ async fn open_channel(
     cln: &Lightningd,
     lnd: &Lnd,
 ) -> Result<()> {
+    debug!(target: LOG_DEVIMINT, "Await block ln nodes block processing");
```

Comment:
Was this refactor needed (moving this block from below) to make things faster or is it just ergonomic?

Reply:
This exact like is just a casual shaving-off couple of milliseconds only because it was easy.